### PR TITLE
feat(gui): up to 6 additional domains with 256 char length 

### PR DESCRIPTION
**Pull request by nikolagigic**
Ticket: #23544

### CODE REVIEW

File: src/middlewared/middlewared/plugins/network.py [hunk 1]
```diff
@@ -480,7 +480,9 @@ class DNSService(Service):
         else:
             gc = await self.middleware.call('datastore.query', 'network.globalconfiguration', None, {'get': True})
             if gc['gc_domain']:
-                domain = gc['gc_domain']
+                domains.append(gc['gc_sdomain'])
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/network.py]: I believe you have no tested this again. There is a typo there.

Comment (nikolagigic) [inline src/middlewared/middlewared/plugins/network.py]: I did, I accidently added that 's' in pycharm while commiting
I will fix it

**Comment by william-gr** [comment]
Jenkins: retest this please

File: gui/network/migrations/0006_globalconfiguration_gc_domains.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    operations = [
+        migrations.AddField(
+            model_name='GlobalConfiguration',
+            name='gc_domains',
+            field=models.TextField(
+                max_length=256,
+                verbose_name="Additional domains",
+                default='',
+                blank=True,
+                help_text="Delimit domains by spaces."
```
Comment (wblock) [inline gui/network/migrations/0006_globalconfiguration_gc_domains.py]: "Additional domains to be searched can be entered here, separated by spaces. Be aware that adding search domains can cause slow DNS lookups."

File: gui/network/models.py [hunk 3]
```diff
@@ -70,6 +70,13 @@ class GlobalConfiguration(Model):
             regex=r'^[a-zA-Z\.\-\_0-9]+$',
         )],
     )
+    gc_domains = models.TextField(
+        max_length=256,
+        verbose_name=_("Additional domains"),
+        default='',
+        blank=True,
+        help_text=_("Delimit domains by spaces.")
```
Comment (wblock) [inline gui/network/models.py]: "Additional domains to be searched can be entered here, separated by spaces. Be aware that adding search domains can cause slow DNS lookups."

Comment (wblock) [inline gui/network/models.py]: This is a clearer explanation and adds the warning that extra search domains can slow down DNS lookups.

**Comment by kmoore134** [comment]
Jenkins: retest this please

**Comment by kmoore134** [comment]
Jenkins: retest this please

**Comment by kmoore134** [comment]
Jenkins: retest this please CLEAN

**Pull request merged**